hbase-asyncfs/src/main/java/org/apache/hadoop/hbase/io/asyncfs/FanOutOneBlockAsyncDFSOutputHelper.java

```diff
@@ -620,4 +622,17 @@ static void sleepIgnoreInterrupt(int retry) {
     } catch (InterruptedException e) {
     }
   }
+
+  public static String getDataNodeInfo(Set<DatanodeInfo> datanodeInfos) {
```

Comment:
Just declare the parameter as Collection<DatanodeInfo>, or even use Stream<DatanodeInfo>, so we do not lose the order when get the string description for an array?

Reply:
OK, It has been changed to Collection